os/mm/umm_heap/umm_malloc.c

```diff
@@ -113,13 +113,13 @@ void *malloc_at(int heap_index, size_t size)
 	ARCH_GET_RET_ADDRESS
```

Comment:
Please add checking size for malloc_at also, `if (size == 0)`.

Reply:
Added.